src/modules/translation/languages/en.json

```
@@ -206,10 +206,15 @@
     }
   },
   "create_multiple_items_modal": {
-    "title": "Create multiple items",
-    "successful_title": "Items created",
-    "cancelled_title": "Cancelled multiple items creation",
-    "failed_title": "Failed multiple items creation",
+    "create_title": "Create multiple items",
+    "edit_title": "Edit items in bulk",
+    "edit_subtitle": "In order to edit an exiting item, use the same URN",
+    "create_successful_title": "Items created",
+    "edit_ successful_title": "Items updated",
```

There's a whitespace in the key:
```suggestion
    "edit_successful_title": "Items updated",
```